scipy/stats/_sensitivity_analysis.py

```diff
@@ -378,22 +385,24 @@ def sobol_indices(
 
     :math:`S_{i}` corresponds to the first-order term which apprises the
     contribution of the i-th parameter, while :math:`S_{ij}` corresponds to the
-    second-order term which informs about the correlations between the
+    second-order term which captures the contribution of interactions between the
     i-th and the j-th parameters. These equations can be generalized to compute
     higher order terms; however, they are expensive to compute and their
     interpretation is complex.
     This is why only first order indices are provided.
 
-    Total indices represent the global contribution of the parameters on the
-    variance of the QoI and are defined as:
+    Total order indices represent the total contribution to the variance of the QoI
+    of each parameter (including all its interactions, of any order, with other parameters),
+    and are defined as:
 
     .. math::
 
         S_{T_i} = S_i + \sum_j S_{ij} + \sum_{j,k} S_{ijk} + ...
         = 1 - \frac{\mathbb{V}[\mathbb{E}(Y|x_{\sim i})]}{\mathbb{V}[Y]}.
 
-    First order indices sum to 1, while the sum of total order indices will be
-    greater than 1 if there are interactions.
+    First order indices sum to at most 1, while total order indices sum to at least 1.
```

Comment:
This is too verbose. I would just change "to at most 1". And if we want we can say that "if there are no interactions, the sum or total and first order indices is equal to 1."

Reply:
I'm struggling to see how to make this less verbose without losing information. Maybe just substitute whatever you'd like it to say here?

Reply:
Actually ok now 👍